### ISSUE TITLE
Resnet baseline setup on Stanford Car DataSet

### ISSUE THREAD

**Issue by prateekgupta891** [facebookresearch/FixRes#13]
I wanted to know if this repo can also be used to set up resnet baseline accuracy as mentioned in the paper for stanford car dataset. If the input parameters can somehow be changed for stanford car dataset. 
Highlighting some procedural steps would also be great.
(I have been trying to setup baseline ResNet accuracy on Stanford Car Dataset using multiple repos, but have failed in y attempts any help would highly appreciated.)

**Comment by TouvronHugo** [comment]
Hi @prateekgupta891  ,
Yes, you can totally use this repo for Stanford car dataset. For transfer-learning tasks we do as follows:
(1) we initialize our network with the weights learned on ImageNet;
(2) we train it entirely for several epochs (approximately 50);
(3) we use the FixRes Method.
For other hyperparameters we use the same ones as for ImageNet.
